lib/pagseguro/payment_request.rb

```diff
@@ -82,8 +84,23 @@ def register
       Response.new Request.post("checkout", params)
     end
 
+    # Adds an extra parameter to payment request. It's useful when you need
+    # to send a parameter to api that's not mapped in the gem yet.
+    def add_parameter(name, value)
+      self.extra_params << {name.to_sym => value}
+    end
+
+    # Adds an extra parameter with index.
+    def add_indexed_parameter(name, value, index)
```

Comment:
Esse método deve fazer isso mesmo? Pelo nome, parece que ele adiciona um parâmetro em um indice específico.

Maaaas, se for isso mesmo, da pra fazer `add_parameter("#{name}#{index}", value) só